.github/workflows/deploy-cloud.yaml

```diff
@@ -26,6 +26,7 @@ jobs:
       IS_CLOUD_INSTANCE: true
       POSTHOG_KEY: ${{ secrets.POSTHOG_KEY }}
       POSTHOG_HOST: ${{ secrets.POSTHOG_HOST }}
+      ONBOARDING_USER_JOINED_DATE_THRESHOLD: ${{ secrets.ONBOARDING_USER_JOINED_DATE_THRESHOLD }}
```

Comment:
Is it intentional the date is stored as a secret? Didn't you mean to use environment variables instead? (https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#creating-configuration-variables-for-a-repository)